src/lib.rs

```diff
@@ -4,13 +4,16 @@ use winapi::shared::ntstatus;
 
 use std::fmt;
 
+pub mod asymmetric;
 pub mod buffer;
+pub mod handle;
```

Comment:
I'm not sure helpers is a good idea, since it mostly collects FFI-specific code like inline blobs or window-specific strings. For example, at some point we might even replace WideCString with a the `widestring` crate or from_bytes/as_bytes with the `bytemuck` crate, whereas handle module is only related to CNG.

I'll mark this module as private.